client/components/EditProduct.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react'
 import {ProductForm} from '../components'
+import {updateProductById} from '../store'
+import {connect} from 'react-redux'
 
 class EditProducts extends React.Component {
-  submit = values => {
-    console.log(values)
+  submit = editedProduct => {
+    this.props.updateProductById(editedProduct)
+    this.props.history.push(`/product/${this.props.match.params.productId}`)
   }
   render() {
     return <ProductForm {...this.props} onSubmit={this.submit} />
   }
 }
 
-export default EditProducts
+const mapDispatchToProps = dispatch => ({
+  updateProductById: product => dispatch(updateProductById(product))
+})
+
+export default connect(null, mapDispatchToProps)(EditProducts)
+
+
```

Comment:
Not sure I understand why this component was edited?

These look like old changes.  Try to rebase (pull in commits from master) before you submit a PR next time.
```
$ git pull origin master
(figure out how to merge)
$ git commit (merge changes)
$ git push
```